sync/bug.py

```diff
@@ -28,6 +28,10 @@
 if "REOPENED" not in bugsy.bug.VALID_STATUS:
     bugsy.bug.VALID_STATUS.append("REOPENED")
 
+# Trying to send this field makes bug creation fail
+if "cc_detail" in bugsy.bug.ARRAY_TYPES:
```

Comment:
for my own understanding
https://bugzilla.readthedocs.io/en/latest/api/core/v1/bug.html?highlight=cc_detail#get-bug

> Array of objects containing detailed user information for each of the cc list members. To see the keys included in the user detail object, see below.

Reply:
Yeah, the underlying bug is that bugsy eagerly populates these fields with an empty array, so if you get the field and it wasn't in the response, you get an empty array rather than nothing. But when it comes to creating a bug that approach doesn't work because not all of the fields that are arrays in the response are valid in the request. So the real fix is to avoid sending fields the user didn't explictly set. But the wrokaround is just to not populate this field at all since we don't use it for anything.